client/starwhale/utils/venv.py

```diff
@@ -440,22 +447,25 @@ def venv_activate_render(
     else:
         sw_cntr_content = host_content
 
-    _render_sw_activate(sw_cntr_content, host_content, workdir)
+    _render_sw_activate(sw_cntr_content, host_content, workdir, quiet)
 
 
-def _render_sw_activate(sw_cntr_content: str, host_content: str, workdir: Path) -> None:
+def _render_sw_activate(
+    sw_cntr_content: str, host_content: str, workdir: Path, quiet: bool = False
+) -> None:
```

Comment:
We use `quiet` to tell if the action should break no not when an error occurs:  https://github.com/star-whale/starwhale/blob/ef181f3f83e2a51fe1f8078942c44a1bf5f376b1/client/starwhale/base/tag.py#L60
Will `verbose` be better?

Reply:
fixed